$MirOS: ports/security/nss/patches/patch-security_nss_lib_ckfw_builtins_nssckbi_h,v 1.4 2011/03/30 10:28:11 tg Exp $

	Raise along with DASH_VER in Makefile
	when ports/essentials/ca-certificates/certdata.c changes.

	Sync with: ports/www/firesomething/patches/patch-security_nss_lib_ckfw_builtins_nssckbi_h

--- security/nss/lib/ckfw/builtins/nssckbi.h.orig	Thu May 21 19:50:05 2009
+++ security/nss/lib/ckfw/builtins/nssckbi.h	Sat Sep  3 16:19:38 2011
@@ -76,7 +76,7 @@
  */
 #define NSS_BUILTINS_LIBRARY_VERSION_MAJOR 1
 #define NSS_BUILTINS_LIBRARY_VERSION_MINOR 75
-#define NSS_BUILTINS_LIBRARY_VERSION "1.75"
+#define NSS_BUILTINS_LIBRARY_VERSION "1.75.0.6"
 
 /* These version numbers detail the semantic changes to the ckfw engine. */
 #define NSS_BUILTINS_HARDWARE_VERSION_MAJOR 1
